packages/waku/src/router/define-router.ts

```diff
@@ -152,20 +155,26 @@ export function unstable_defineRouter(
   ) => {
     const pathConfig = await getMyPathConfig();
     const path2moduleIds: Record<string, string[]> = {};
-    for (const { pathname: pathSpec } of pathConfig) {
+
+    const promises = pathConfig.map(async ({ pathname: pathSpec, pattern }) => {
       if (pathSpec.some(({ type }) => type !== 'literal')) {
-        continue;
+        return;
       }
       const pathname = '/' + pathSpec.map(({ name }) => name).join('/');
       const input = getInputString(pathname);
-      const moduleIds = await unstable_collectClientModules(input);
-      path2moduleIds[pathname] = moduleIds;
-    }
+      path2moduleIds[pattern] = await unstable_collectClientModules(input);
```

Comment:
Are we sure if we can run this concurrently?

Reply:
My [last commit](https://github.com/dai-shi/waku/pull/668/commits/ce07555acfbdb465d14b07ac695288f9e517d399) was **not** an answer to this comment. I saw it just now.

But maybe it does prove it? What are your concerns collecting client modules in parallel?

Reply:
I'm not sure what my concerns are, tbh.
(When I coded it, I thought it's safer to make it sequential intentionally. But I don't remember why.)

Reply:
The test are green, and It is about to change to static analysis anyway right?

Reply:
Exactly!

Reply:
And at that point it does not make sense to run it concurrently anyway 😅
I just need it for the POC I'm working on.